src/release_notes_workflow/releasenotes_gitlog_getcommit.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import subprocess
+
+
+class GitLogGetCommit:
```

Comment:
Both methods belong in https://github.com/opensearch-project/opensearch-build/blob/main/src/git/git_repository.py

Reply:
The `gitlog` is only required for specific purpose, here to fetch commitID and date from repo log, but the [git_repository.py](https://github.com/opensearch-project/opensearch-build/blob/main/src/git/git_repository.py), is for generic purposes used across other build code. Hence created as a separate methods inside release notes workflow.